src/about-us/index.html

```diff
@@ -10,14 +10,12 @@
                     block__flush-top">
         <h1>About Us</h1>
         <p class="h3">
-            We're a new government agency created
-            after the financial crisis of 2008.
-            We write and enforce rules to ensure that markets for
-            consumer financial products and services are fair and open,
-            and to make financial information easier to understand.
-            We also build tools and resources that help arm consumers
-            with the information they need,
-            and empower consumers to make the best financial decisions.
+            The Consumer Financial Protection Bureau (CFPB)
+            is a 21st century agency that helps consumer finance markets work
```

Comment:
Technically probably should be `21<sup>st</sup>` if our styles can handle that correctly.